lib/salesforce/donation_validator.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'salesforce/result'
+
+module Salesforce
+  class DonationValidator
+    def self.execute(data, supporter)
+      new(data, supporter).execute
+    end
+
+    def initialize(data, supporter)
+      @data = data
+      @supporter = supporter
+    end
+
+    def execute
+      Result.new(fields, errors)
+    end
+
+    private
+
+    attr_reader :data, :supporter
+
+    def fields
+      return unless errors.empty?
+      {
+        Amount: data.amount.to_s,
```

Comment:
Are all these fields test data to be replaced, or should they extracted to constants?

For example: `STAGE_NAME = 'Received'.freeze` ?

Reply:
This is all hardcoded stuff for the moment, so no need to make them constants.